feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -24,61 +24,128 @@ <h2>{{ 'configurator.overviewForm.noAttributeHeader' | cxTranslate }}</h2>
-            <cx-configurator-overview-attribute
-              [attributeOverview]="attributeOverview"
-            >
-            </cx-configurator-overview-attribute>
-          </ng-container>
+        <span class="cx-visually-hidden">
+          {{
+            'configurator.a11y.group'
+              | cxTranslate
+                : {
+                    group: group.groupDescription
+                  }
+          }}
+        </span>
+        <h2 aria-hidden="true">
+          <span>{{ group.groupDescription }}</span>
+        </h2>
+
+        <div
+          *ngFor="let attributeOverview of group.attributes; let i = index"
+          class="cx-attribute-value-pair"
+          [ngClass]="getStyleClasses(group.attributes, i)"
+        >
+          <ng-container [ngSwitch]="attributeOverview?.type">
+            <ng-container *ngSwitchCase="attributeOverviewType.GENERAL">
+              <cx-configurator-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-overview-attribute>
+            </ng-container>
+
+            <ng-container *ngSwitchCase="attributeOverviewType.BUNDLE">
+              <cx-configurator-cpq-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-cpq-overview-attribute>
+            </ng-container>
 
-          <ng-container *ngSwitchCase="attributeOverviewType.BUNDLE">
-            <cx-configurator-cpq-overview-attribute
-              [attributeOverview]="attributeOverview"
-            >
-            </cx-configurator-cpq-overview-attribute>
+            <ng-container *ngSwitchDefault>
+              <cx-configurator-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-overview-attribute>
+            </ng-container>
           </ng-container>
+        </div>
+      </div>
+      <ng-container *ngIf="group.subGroups?.length > 0">
+        <ng-container
+          *ngTemplateOutlet="
+            groups;
+            context: { overviewGroups: group.subGroups, level: level + 1 }
+          "
+        ></ng-container>
+      </ng-container>
+    </ng-container>
+  </ng-container>
 
-          <ng-container *ngSwitchDefault>
-            <cx-configurator-overview-attribute
-              [attributeOverview]="attributeOverview"
-            >
-            </cx-configurator-overview-attribute>
+  <ng-container *cxFeatureLevel="'6.0'">
+    <ng-container *ngFor="let group of overviewGroups">
+      <div
+        class="cx-group"
+        [class.topLevel]="level === 1"
+        [class.subgroupTopLevel]="level === 1 && group.subGroups?.length > 0"
+        [class.subgroup]="level > 1"
+        [class.subgroupLevel2]="level === 2"
+        [class.subgroupLevel3]="level === 3"
+        [class.subgroupLevel4]="level === 4"
+        [class.subgroupLevel5]="level === 5"
```

Comment:
Can we make the assignment of the subgrouplevel class as a method?
This would make it easier to change the amount of subgroups that get stacked.

Reply:
Extracted the style class determination to a method.